.github/workflows/build-system.yml

```diff
@@ -54,7 +56,11 @@ jobs:
       - uses: novelrt/setup-cpp@v1
       - name: Build and Test NovelRT - ${{ matrix.configuration }}
         id: build
-        run: conan build . --build-folder build
+        run: |
+          ${{ matrix.c }} ${{ matrix.cxx }} cmake -S . -B build -DCMAKE_BUILD_TYPE=${{ matrix.configuration }}
+          cmake --build build --config ${{ matrix.configuration }}
+          cd build/tests/NovelRT.Tests
+          ctest
```

Comment:
Just calling out: Conan was having issues running tests (it was invoking Run_Tests.vcxproj???), so I've switch to CMake builds only (as we're moving from Conan anyways).

Also, if our matrix doesn't have a definition for `c` or `cxx` (i.e. Windows or macOS currently), it just puts a space and shouldn't break the command.

Reply:
There's likely a better way to do this, but for now I'm fine with leaving this just to get the gtest stuff in.